misc.md

```diff
@@ -0,0 +1,41 @@
+## [Shading](https://github.com/rallyhealth/sbt-shading) + [SemVer](https://semver.org/) + [MiMa](https://github.com/lightbend/mima)
+
+Many of Rally Health's libraries need to work with JSON. However, we have
+found (through long painful experience) that letting them use Play-Json
+puts us into a particular form of Dependency Hell: the libraries wind up
+dependent on a *specific version* of Play-Json, which means that upgrading
+Play requires upgrading much of our ecosystem, which is a hassle. We want
+to decouple our libraries from Play as much as possible, to reduce this
+friction.
+
+So we are encouraging libraries to make use of weePickle instead: it's
```

Comment:
I _think_ you mean `uPickle` here. Then on line 19, to be clear, you can say, `So this, weePickle, is a shaded fork of uPickle`.

Reply:
become an early contributor - put up a PR to make the changes that you want to see :)